tests/integration/method-returns-fieldlist/expected/logs/warning.log

```diff
@@ -0,0 +1 @@
+app.WARNING: The definition of a return description as field list is deprecated.
```

Comment:
The field list syntax in rst: https://www.sphinx-doc.org/en/master/usage/restructuredtext/field-lists.html

Reply:
Ah, so the problem is that the `:return:` is not immediately after `php:method`?

Reply:
yes in that case we have to parse the content instead of just asking for an option. Also it is counter-intuitive because all the other options belong to the method. However sphinx is currently doing it like this

Reply:
Ack